public/index.html

```diff
@@ -8,12 +8,12 @@
 	 <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
```

Comment:
why did you remove font-awesome link?

Reply:
I removed the font awesome cdn link because of it was being repeated in the index.html file, so I kept one instead.